moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h

```diff
@@ -109,6 +109,9 @@ class JogCalcs
 
   void insertRedundantPointsIntoTrajectory(trajectory_msgs::JointTrajectory& trajectory, int count) const;
 
+  // Remove the Jacobian row and the delta-x element of one Cartesian dimension, to take advantage of task redundancy
+  inline void removeDimension(Eigen::MatrixXd& matrix, Eigen::VectorXd& delta_x, const int row_to_remove);
```

Comment:
my understanding is that using inline without inlining the function in the header file, does nothing in C++

Also, its a medium size function, i think you should remove inline all together.